solution/AddTest/office-test.js

```diff
@@ -7,24 +7,26 @@ describe('office', function () {
             const OFFICE = {
                 id: 1423,
                 name: "Bordeaux",
-                address: {
-                    street: "39 Rue du Château d'Eau",
-                    city: "Bordeaux", country: "France", zip: "33000",
-                },
-                drivers: [{
-                    id: 3, vehicle: 'renault',
-                    first: 'Daniel', last: 'RICCARDO',
-                    seniority: 5, age: 30,
-                }, {
-                    id: 33, vehicle: 'Honda',
-                    first: 'Max', last: 'VERSTAPPEN',
-                    seniority: 3, age: 22,
-                }, {
-                    id: 14, vehicle: 'mercedes',
-                    first: 'Lewis', last: 'HAMILTON',
-                    seniority: 12, age: 34,
-                },
-                ]
+                meta: {
+                    address: {
+                        street: "39 Rue du Château d'Eau",
+                        city: "Bordeaux", country: "France", zip: "33000",
+                    },
+                    drivers: [{
+                        id: 3, vehicle: 'renault',
+                        first: 'Daniel', last: 'RICCARDO',
+                        seniority: 5, age: 30,
+                    }, {
+                        id: 33, vehicle: 'Honda',
+                        first: 'Max', last: 'VERSTAPPEN',
+                        seniority: 3, age: 22,
+                    }, {
+                        id: 14, vehicle: 'mercedes',
+                        first: 'Lewis', last: 'HAMILTON',
+                        seniority: 12, age: 34,
+                    },
+                    ]
+                }
```

Comment:
ℹ️ Note: Just wrapping everything in a `meta` property. The `address` and `drivers` values are unchanged here.